README.md

```diff
@@ -1,9 +1,9 @@
 # AML Dataset implementation guide
 
-### Prerequisites:
-- a  Neo4j 3.5.x server with read/write access
+### Prerequisites: 
+- a  Neo4j 4.2 or greater server with `admin` or `architect` access rights
 - APOC procedures are enabled in Neo4j (more information [here](https://neo4j.com/docs/labs/apoc/current/introduction/#installation))
-- a  Linkurious Enterprise 2.9.x server with admin access ([getting started guide](https://doc.linkurio.us/admin-manual/latest/getting-started/))
+- a Linkurious Enterprise 3.0 or greater with `admin` access rights ([getting started guide](https://doc.linkurio.us/admin-manual/latest/getting-started/))
```

Comment:
I'll put 3.0.4 or greater (maybe higher is a better word?) since we changed the security of alerts

Reply:
done